spec/formatting.md

```diff
@@ -695,14 +695,23 @@ These are divided into the following categories:
     > when * {Value is not one}
     > ```
     >
+    > Example valid messages that do not result in a _Missing Selector Annotation error_:
```

Comment:
As above...

```suggestion
> By contrast, here are some valid messages that do not result in a _Missing Selector Annotation error_:
```

Is it that helpful to show valid messages? Shouldn't we focus on triggering behavior?

Reply:
I'm not sure how to clarify what "does _not_ directly or indirectly reference an expression with an annotation" (emphasis added) means without contrasting it with a message that _does_ indirectly reference an expression with an annotation. Do you have any suggestions for how to make that language clearer without using a valid example?

Reply:
I thought of another way to phrase the condition for validity that didn't require a valid example to explain -- let me know what you think.

Reply:
I haven't looked at the proposed text yet, but I will note: examples are good and even if we think of a better way of explaining something, we should be looking for opportunities to incorporate examples. My above comment might go towards some style guide activity, and, in a more general way, start to build some infrastructure to support conversion of our spec into a proper HTML document (I'll create a task for myself to capture).